_hacks/10.md

```diff
@@ -0,0 +1,14 @@
+---
+hack_number: 10
+date: 2021-01-30
+happened: true
+---
+We had our most _international_ day of remote hacks yet, with participants California and Munich!
+
```

Comment:
Participants in/from?

Reply:
You and your fancy prepositions! Right I'll fix that, I may have been mashing the keyboard too hard..